### PR TITLE
feat: add Faker and change populate.py

### PR DESCRIPTION
**Pull request by developerfred**
#1342

https://faker.readthedocs.io/en/master/fakerclass.html#examples

### CODE REVIEW

**Comment by ColdHeat** [comment]
Wow fast PR! Can you move the Faker dep to `development.txt` instead of `requirements.txt` though?

**Comment by ColdHeat** [comment]
Also looks like the last version to support Py2 was 3.0.1 so please pin that instead since Py2 is still supported by CTFd. 
https://github.com/joke2k/faker/blob/master/CHANGELOG.rst#301---14-january-2020

**Comment by ColdHeat** [comment]
The emails you're generating appear to be invalid. 

image

File: populate.py [hunk 1]
```diff
@@ -1177,35 +199,35 @@
 
 
 def gen_sentence():
-    return " ".join(random.sample(lorems, 50))
+    return fake.text()
 
 
 def gen_name():
-    return random.choice(names)
+    return fake.name()
```
Comment (ColdHeat) [inline populate.py:206]: Can you make this return a first name only inline with previous behavior?

Comment (developerfred) [reply populate.py:206]: yes

Comment (developerfred) [reply populate.py:206]:
```
def gen_name():
    from random import shuffle, seed
    from faker.providers.person.en import Provider

    first_names = list(set(Provider.first_names))
    seed(1551)
    shuffle(first_names)
    
    return first_names[0:1000]
```

with this new script I can print a list of 1000 first names without repetition.

File: populate.py [hunk 2]
```diff
@@ -1177,35 +199,42 @@
 
 
 def gen_sentence():
-    return " ".join(random.sample(lorems, 50))
+    return fake.text()
 
 
 def gen_name():
-    return random.choice(names)
+    from random import shuffle, seed
```
Comment (ColdHeat) [inline populate.py:206]: This seems odd, do you need to do all of this for each function call? Is there not a direct method in Faker to generate a random first name?

File: requirements.txt [hunk 3]
```diff
@@ -26,4 +26,4 @@ pathlib2==2.3.5
 flask-marshmallow==0.10.1
 marshmallow-sqlalchemy==0.17.0
 boto3==1.10.39
-marshmallow==2.20.2
+marshmallow==2.20.2
```
Comment (ColdHeat) [inline requirements.txt:29]: You shouldn't have any changes in this file, can you revert this?

File: populate.py (hunk 2 above)
Comment (developerfred) [reply populate.py:206]: exists but can repeat itself at some point

File: requirements.txt (hunk 3 above)
Comment (developerfred) [reply requirements.txt:29]: yeap

**Comment by developerfred** [comment]
@ColdHeat what command do i use to remove the change in the requirements of my pull request?

**Pull request merged**